apps/db_assembler/README.md

```diff
@@ -1,5 +1,7 @@
 ## Description
 
+**ATTENTION:** this tool is not supported anymore. It has been replaced by the **dbimport** tools integrated in AC server sources
+
```

Comment:
shall we add also a warning when someone tries to use it? not necessarily in this PR